packages/property-grid/src/components/ElementList.tsx

```diff
@@ -4,7 +4,7 @@
 *--------------------------------------------------------------------------------------------*/
 
 import * as React from "react";
-import * as classnames from "classnames";
```

Comment:
@aruniverse - Since you are messing with classnames you can update to latest package "classnames": "^2.3.1", and remove the related types files as the new package delivers types.  That is what I did in imodeljs.